core/src/main/java/com/linecorp/armeria/internal/metric/RequestMetricSupport.java

```diff
@@ -101,6 +102,25 @@ private static void onResponse(RequestLog log, MeterIdPrefixFunction meterIdPref
                                                                      ClientRequestMetrics.class,
                                                                      DefaultClientRequestMetrics::new);
         updateMetrics(log, metrics);
+        final ClientConnectionTimings timings = ClientConnectionTimings.get(log);
+        if (timings != null) {
+            metrics.connectionAcquisitionDuration().record(
+                    timings.connectionAcquisitionDurationNanos(),
```

Comment:
nit: This line might be moved to the above line in order to beautify the code. 😉